api/src/main/java/org/eclipse/microprofile/reactive/streams/ConnectingOperators.java

```diff
@@ -33,32 +33,32 @@
     /**
      * Connect the outlet of the {@link Publisher} built by this builder to the given {@link Subscriber}.
      *
-     * @param subscriber The subscriber to connect.
+     * @param subscriber The subscriber to connect, it should not be reused. 
```

Comment:
As with the comment on `CompletionSubscriber`, the gotcha here is not so much about the subscriber passed in, I don't think it's going to be common that people will pass a single subscriber to multiple `to` methods. Rather, it will be that the resulting `SubscriberBuilder` or `CompletionRunner` will only be able to be built/run once, if it's built/run multiple times, only the first one will work as expected, subsequent ones will, in accordance with the Reactive Streams spec, be immediately cancelled.

Reply:
I have tried to word this better in [a646f62](https://github.com/eclipse/microprofile-reactive-streams/pull/110/commits/a646f622b20699ddb4521e12ec03413ae63dcfc6)